build.gradle

```diff
@@ -716,10 +716,10 @@ subprojects {
             ]
         }
 
-        // Create an allTests task per project
-        tasks.create(name: 'allTests', type: GradleBuild, group: 'Verification') {
-            description 'Run all Checker Framework tests'
-            tasks = ['test', 'checkPurity']
+        // Create an nonCheckerJunitTests task per project
```

Comment:
I think a comment, stating that for all gradle projects other than `checker` this is identical to allTests, would be helpful.  (A reader can figure it out, but saying it explicitly would be helpful.)